src/js/jsx/DocumentHeader.jsx

```diff
@@ -167,6 +193,18 @@ define(function (require, exports, module) {
                     <div className="document-header" ref="tabContainer">
                             {documentTabs}
                     </div>
+                    <div className="generator-header">
+                        <div className="generator-header-buttons">
+                            <Button
+                                className="button-plus button-simple"
+                                title={strings.TOOLTIPS.EXPORT_DIALOG}
+                                disabled={exportDisabled}
+                                onClick={this._openExportPanel}>
+                                <SVGIcon
```

Comment:
thanks, I like this better as a Button (even though  you have to deal with SVGIcon separately ... and we might oughta make Button handle this for us like SplitButton does) ... but I'm just rambling.  Ignore all this.